moveit_core/planning_scene/test/test_multi_threaded.cpp

```diff
@@ -76,6 +76,8 @@ class CollisionDetectorThreadedTest : public testing::Test
     robot_model_ok_ = static_cast<bool>(robot_model_);
```

Comment:
Please replace this line with your assertion.
The following statements actually already rely on a valid robot_model_.

Reply:
You're right, changed it.